src/courseware/course/sidebar/SidebarTriggers.jsx

```diff
@@ -13,13 +13,13 @@ const SidebarTriggers = () => {
   const isMobileView = useWindowSize().width < breakpoints.small.minWidth;
 
   return (
-    <div className="d-flex ml-auto">
+    <div className="d-flex ml-auto gap-3" style={{ gap: '10px' }}>
```

Comment:
kindly add to the brand-openedx package and use a class name instead

```suggestion
    <div className="d-flex ml-auto name-of-class-goes-here">
```